### ISSUE TITLE
Does distance to end of transcript make sense?

### ISSUE THREAD

**Issue by Shians** [LuyiTian/scPipe#70]
I can't really tell what the point of the YE is, [this](https://github.com/LuyiTian/scPipe/blob/master/R/wrapper_scPipeCPP.R#L134) says that it's "mapping status tag". Looking into the code it looks like this is the "distance to end of transcript" which is calculated without introns.

Since I've disabled the code to calculate the distance it's always -9999999 (the initialised value). I'm not sure if it even makes sense to calculate the distance to end of transcript, because splice variants mean that it's often impossible to tell which transcript you got, I don't think this value is relevant anywhere downstream either.

So I think we either delete all references to this or state more clearly exactly what this value is and how it's calculated.

**Comment by LuyiTian** [comment]
more than 90% of the read located in 0-1000bp upstream of the transcript end site from all the data I have analyzed. so that distance should make sense. it will just be a rough estimation though. polyA end position is a complicated issue and I don't want to calculate it in exon_mapping, should probably put a new function for that.

**Comment by LuyiTian** [comment]
yes, but it is minor. actually, most of the genes don't have such isoforms, so this value will be a rough estimate with high false positive, but false negative will be low.
